tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -7133,6 +7133,30 @@ declare namespace ts.server.protocol {
          */
         body?: string[];
     }
+    /**
+     * A request to get encoded semantic classifications for a span in the file
+     */
+    interface EncodedSemanticClassificationsRequest extends FileRequest {
```

Comment:
Do we also need to add the response types? We currently have to maintain a copy of them in VS Code: https://github.com/microsoft/vscode/blob/3d500ebd8b4a919e8631e916f417e5c68eb90d3f/extensions/typescript-language-features/src/languageFeatures/semanticTokens.ts#L275

Reply:
Aren’t they in typescript/tsserver .d.ts ?

Reply:
We currently only use `protocol.d.ts` with VS Code. I think this should have all the types used communicating from or to tsserver

Reply:
Makes sense, added